src/rp2_common/hardware_i2c/i2c.c

```diff
@@ -130,7 +130,7 @@ void i2c_set_slave_mode(i2c_inst_t *i2c, bool slave, uint8_t addr) {
     i2c->hw->enable = 1;
 }
 
-static int i2c_write_blocking_internal(i2c_inst_t *i2c, uint8_t addr, const uint8_t *src, size_t len, bool nostop,
+static i2c_result_t i2c_write_blocking_internal(i2c_inst_t *i2c, uint8_t addr, const uint8_t *src, size_t len, bool nostop,
```

Comment:
For consistency with function-names and return-types, I wonder if `i2c_write_blocking_internal` ought to be renamed to `i2c_write_result_blocking_internal` ? :thinking:  (It's an internal-only function name, so I think we're free to rename it?)

Reply:
out of date